project_template/README.md

```diff
@@ -31,9 +31,9 @@ Prettier can also be run manually using `yarn run prettier` or `yarn run prettie
 
 ### General
 
-* Ruby + Bundler, for the integration specs
 * Node.js >=6 + NPM, for the build system
 * Chrome v59.x or higher to run the tests headless. You might need install it from the [beta channel](https://www.google.com/chrome/browser/beta.html).
+* Java 8 (or higher) to run the end-to-end tests (it powers the selenium server)
```

Comment:
When I was playing with Nightwatch I used Chromedriver directly. That way we don't have a dependency on Java/Selenium.